mms/protocol/otf_message_handler.py

```diff
@@ -123,9 +123,6 @@ def create_load_model_response(code, message):
 
 
 def _retrieve_buffer(conn, length):
-    if length > MAX_BUFFER_SIZE:
```

Comment:
Should we remove MAX_BUFFER_SIZE as well?

Reply:
Yes. I will remove it.